### PR TITLE
Using the Feature Store with Databricks and S3

### PR DESCRIPTION
**Pull request by steffengr**

### CODE REVIEW

**Comment by tkakantousis** [comment]
`This requires a Hopsworks instance that is available from Databricks`
do you mean `accessible`?

**Comment by tkakantousis** [comment]
`setup_databricks only needs to be executed onces per cluster`
once

**Comment by tkakantousis** [comment]
We'll use numpy and pandas for data generation, pyspark for feature engineering, tensorflow and keras for model training, and the hops `featurestore` library for interacting with the feature store.

we use lower case because we refer to the python modules instead of the framework names?

**Comment by tkakantousis** [comment]
`display(dbutils.fs.ls(MOUNT_NAME))`
does this shell have an output?

**Comment by tkakantousis** [comment]
`The feature store has an abstraction of a training dataset, which is a dataset with a set of features (potentially from many different feature groups) and labels (in case of supervised learning).`

I would add that Hopsworks generates the training dataset in the file format chosen by the data scientists and needed by the ML framework to be used in training
